setup.cfg

```diff
@@ -26,7 +29,7 @@ platform = any
 
 url = https://github.com/Iotic-Labs/iotics-identity-py
 keywords = iotics, did, decentralised identity, decentralized identity, digital twin
-python_requires = >=3.7
+python_requires = >=3.8
```

Comment:
3.7 has been flagged as EOF on June 2023 https://devguide.python.org/versions/
It is too early to remove the support.
I had a quick chat with Lorenzo and for now, they are expecting the client to use a version not older than 3.7.

Reply:
Just to clarify: doing the update is necessary but we should make delivery aware of the change in advance and give them some time to communicate with the users

Reply:
Support for 3.7 already ended, we should not be using it anymore - there are far too many companies using unsupported versions :(